webview/src/experiments/components/emptyState/GetStarted.tsx

```diff
@@ -1,8 +1,34 @@
 import React from 'react'
 import { Welcome } from './Welcome'
 import { AddColumns } from './AddColumns'
+import { RemoveFilters } from './RemoveFilters'
 import { EmptyState } from '../../../shared/components/emptyState/EmptyState'
 
-export const GetStarted: React.FC<{ showWelcome: boolean }> = ({
-  showWelcome
-}) => <EmptyState>{showWelcome ? <Welcome /> : <AddColumns />}</EmptyState>
+export const GetStarted: React.FC<{
+  hasColumns: boolean
+  hasOnlyDefaultColumns: boolean
+  hasRows: boolean
+  hasFilters: boolean
+}> = ({ hasColumns, hasFilters, hasOnlyDefaultColumns, hasRows }) => {
+  if (hasColumns && hasOnlyDefaultColumns) {
+    return (
+      <EmptyState>
+        <AddColumns />
+      </EmptyState>
+    )
+  }
+
+  if (!hasRows && hasFilters) {
+    return (
+      <EmptyState>
+        <RemoveFilters />
+      </EmptyState>
+    )
+  }
+
+  return (
+    <EmptyState>
+      <Welcome />
```

Comment:
[F] I don't think we should ever get here anymore (mainly because of the setup webview) but will leave "just in case"